doc/source/rllib/package_ref/index.rst

```diff
@@ -0,0 +1,21 @@
+.. include:: ../we_are_hiring.rst
+
+.. _rllib-reference-docs:
+
+RLlib API Reference
+===================
+
+.. tip:: We'd love to hear your feedback on using RLlib - `ask your questions or give feedback at <https://discuss.ray.io>`_!
```

Comment:
this part doesn't render properly btw

Reply:
Looks actually ok on my laptop.

<img width="849" alt="image" src="https://user-images.githubusercontent.com/5839854/142594076-253ec456-38c3-494a-b84c-d057f0e7a5c3.png">

Reply:
Great catch. href doesn't get rendered, of course. Fixed